horace_core/sqw/file_io/@faccess_sqw_v3/put_bytes.m

```diff
@@ -0,0 +1,26 @@
+function obj = put_bytes(obj, to_write)
```

Comment:
`put` where? I would favour `write` or `output` but maybe that's me. Particularly as you "write_v2_obj_header"

Reply:
`put_` seems to be the common prefix for public functions that write to file in this class (`put_sqw`, `put_dnd`, `put_headers` etc). 

I think for consistency's sake, it may be better to rename `write_v2_obj_header_` → `put_v2_obj_header_`.